node/network/bridge/src/lib.rs

```diff
@@ -1287,6 +1287,7 @@ mod tests {
 			self.send_network_event(NetworkEvent::NotificationStreamOpened {
 				remote: peer,
 				protocol: peer_set.into_protocol_name(),
+				negotiated_fallback: None,
```

Comment:
```
    --> node/network/bridge/src/lib.rs:1290:5
     |
1290 |                 negotiated_fallback: None,
     |                 ^^^^^^^^^^^^^^^^^^^ `sc_network::Event::NotificationStreamOpened` does not have this field
```

https://gitlab.parity.io/parity/polkadot/-/jobs/911641#L398

Reply:
Looks like Cargo.lock should be updated

Reply:
The Cargo.lock doesn't need to be updated for companion PRs. It's the Substrate CI (in https://github.com/paritytech/substrate/pull/8682) that automatically puts the fork of Substrate into Polkadot then builds it.
The companion PR, however, needs an approval.